src/lib.rs

```diff
@@ -34,6 +34,12 @@ impl Fernet {
         })
     }
 
+    pub fn generate_key() -> String {
+        let mut key: [u8; 32] = Default::default();
+        rand::OsRng::new().unwrap().fill_bytes(&mut key);
```

Comment:
The unwrap never fails here? They always make me a bit nervous.

Reply:
If its rare enough that it might fail, perhaps the doc string should reflect that it could panic in edge cases.

Reply:
On macOS, FreeBSD, OpenBSD, and Windows it's 100% infallible. On Linux it would fail if `getrandom` wasn't available _and_ you couldn't open `/dev/urandom`. For all intents and purposes I'd consider this a "never" situation.